DependencyInjection/Configuration.php

```diff
@@ -963,4 +1062,119 @@ public function getConfigTreeBuilder()
 
         return $treeBuilder;
     }
+
+    private function handlerTypeAcceptedOptionsValidation(array $v)
+    {
+        if (!array_key_exists('type', $v)) {
+            return $v;
```

Comment:
I think this should rather throw if type is missing? AFAIK there is no valid handler config without a type. Same below if the type is unknown I'd say throw instead of returning a valid value

Reply:
By returning `$v`, I let other validations to take place. Here I'm validating only if handler type has correct options, so I think it pretty safe to return `$v` directly. WDYT?